src/app/utils/MGPSet.ts

```diff
@@ -89,6 +94,9 @@ export class MGPSet<T extends Comparable> implements ComparableObject {
     public isEmpty(): boolean {
         return this.values.length === 0;
     }
+    public isNotEmpty(): boolean {
+        return this.isEmpty() === false;
+    }
```

Comment:
isFilled ?

Reply:
errr non, `containsSomething` ?

Reply:
non "contains something" ça sonne vernaculaire, filled c'est l'inverse de empty, ça ou hasElement(s) je préfère au taPasUnTrucFrère?() que tu suggère :D !

Reply:
Mais non filled c'est pas l'inverse d'empty, c'est son opposé ! `not(empty) !== filled`, un verre à moitié vide n'est pas vide mais n'est pas un verre rempli ! L'inverse de vide c'est qui contient quelque chose ! Donc changé pour `hasElement` yep

Reply:
Donc "rempli à 75%" n'existe pas et "rempli à 100%" est comme dire "au jours d'aujourd'hui" :o 

J'ai check java, ils ont fait simple, y a isEmpty() et pas le reste. 
Là du coup on a moins de négations mais on a hasElements (au pluriel) qui dit oui quand il n'y a que UN élément... langage-wise, c'est correct ? "As tu des bananes ? Oui ! Combien ? Une" ??
Je me dit que p-e juste isEmpty() === false est préférable sinon et que mes OCDs ont p-e été contre-productif là :D

Reply:
oui je pense pas qu'il y a beaucoup de langages/API qui font du hasSomething pour isEmpty() == false !
Langagièrement c'est bon « as-tu des bananes » « oui une ». Mais aussi « as-tu une banane » « oui, deux ».
Mais oui je crois que cette PR a été fort orientée OCD! Du coup je revert celui-ci à isEmpty == false ?